diff --git i/lua/avante/sidebar.lua w/lua/avante/sidebar.lua
index 7b5db64..e254a55 100644
--- i/lua/avante/sidebar.lua
+++ w/lua/avante/sidebar.lua
@@ -1472,7 +1472,7 @@ local base_win_options = {
   wrap = false,
   cursorline = false,
   fillchars = "eob: ",
-  winhighlight = "CursorLine:Normal,CursorColumn:Normal,WinSeparator:"
+  winhighlight = "CursorLine:Normal,WinSeparator:"
     .. Highlights.AVANTE_SIDEBAR_WIN_SEPARATOR
     .. ",Normal:"
     .. Highlights.AVANTE_SIDEBAR_NORMAL,
diff --git i/lua/avante/ui/prompt_input.lua w/lua/avante/ui/prompt_input.lua
index 4993385..4d6bffc 100644
--- i/lua/avante/ui/prompt_input.lua
+++ w/lua/avante/ui/prompt_input.lua
@@ -116,13 +116,11 @@ function PromptInput:open()
   self.winid = winid
 
   api.nvim_set_option_value("wrap", false, { win = winid })
-  api.nvim_set_option_value("winblend", 5, { win = winid })
   api.nvim_set_option_value(
     "winhighlight",
     "FloatBorder:AvantePromptInputBorder,Normal:AvantePromptInput",
     { win = winid }
   )
-  api.nvim_set_option_value("cursorline", true, { win = winid })
   api.nvim_set_option_value("modifiable", true, { buf = bufnr })
 
   local default_value_lines = {}
